base_station/gui/src/components/PidTune.vue

```diff
@@ -63,7 +65,7 @@
         Kp: 0.0,
         Ki: 0.0,
         Kd: 0.0,
-        Kf: 0.0,
+        pidMode: false, // true = sa, false = ra
```

Comment:
Making this a boolean makes it sound like PID control is on or off, rather than switching between two configurations. Is there a better way to name this (i.e., `pidConfig`?) and make it something like an int value to represent state rather than a boolean value? (I don't know JS so if you disagree or I'm misunderstanding we can talk about it)